section-parsers.go

```diff
@@ -176,6 +176,7 @@ func getGlobalParser() *Parsers {
 		&parsers.LogSendHostName{},
 		&parsers.LuaLoad{},
 		&parsers.ConfigSnippet{},
+		&simple.Time{Name: "hard-stop-after"},
```

Comment:
Snippet need to be the last one, please move this in front of parsers.Log{} parser

Reply:
fixed 👍